.github/workflows/precommit.yml

```diff
@@ -1,12 +1,12 @@
 name: Gradle Precommit
 on: [pull_request]
- 
+
 jobs:
   precommit:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest] 
+        os: [windows-latest, macos-latest]
```

Comment:
Maybe add a comment here? `# precommit on ubuntu-latest is run as a part of the gradle-check workflow`